redash/models.py

```diff
@@ -571,6 +581,9 @@ def pre_save(self, created):
         self.query_hash = utils.gen_query_hash(self.query)
         self._set_api_key()
 
+        if not self.access_groups:
+            self.access_groups = self.DEFAULT_ACCESS_GROUPS
```

Comment:
I just tested and the default value works for new objects and also in the migration. Can you verify that you have the correct version of peewee (2.4.7)?